code/modules/cargo/packs.dm

```diff
@@ -147,11 +147,27 @@
 	name = "NULL_ENTRY"
 	hidden = TRUE
 	cost = 14000
-	contains = list(/obj/item/weapon/storage/box/syndicate)
+	contains = null
 	crate_name = "emergency crate"
 	crate_type = /obj/structure/closet/crate/internals
 	dangerous = TRUE
 
+/datum/supply_pack/emergency/syndicate/fill(obj/structure/closet/crate/C)
+	var/list/uplink_items = get_uplink_items()
```

Comment:
See my comment above for anyone unsure of what will actually be included currently.